radix-engine-interface/src/api/types/re_node.rs

```diff
@@ -212,6 +212,7 @@ impl Into<ResourceAddress> for GlobalAddress {
     ScryptoDecode,
 )]
 pub enum NodeModuleId {
+    TypeInfo,
```

Comment:
Can we start with specific `PackageTypeInfo`  and consider generalizing later? I'm adding `ComponentTypeInfo` in another PR.

Reply:
Sounds good